src/ert/scheduler/job.py

```diff
@@ -223,17 +223,25 @@ async def _verify_checksum(self, timeout: int = 120) -> None:  # noqa: ASYNC109
                 logger.error(f"Disk synchronization failed for {file_path}")
 
     async def _handle_finished_forward_model(self) -> None:
-        callback_status, status_msg = await forward_model_ok(self.real.run_arg)
-        if self._callback_status_msg:
-            self._callback_status_msg = status_msg
-        else:
-            self._callback_status_msg += f"\nstatus from done callback: {status_msg}"
-
-        if callback_status == LoadStatus.LOAD_SUCCESSFUL:
-            await self._send(JobState.COMPLETED)
-        else:
-            assert callback_status == LoadStatus.LOAD_FAILURE
-            await self._send(JobState.FAILED)
+        if self._scheduler._fmok_lock:
```

Comment:
You can do this with `async with self._scheduler._fmok_lock:`

Reply:
No, I can't unfortunately. This would need to be mocked for many tests as fmok_lock can be `None`